online-shop/src/app/modules/shopping-cart/components/presentational/shopping-cart-details-view/shopping-cart-details-view.component.scss

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
```

Comment:
As mentioned earlier, those css classes can be refactored into the global styling file. If you need custom or specific styling properties, you can define other classes here

Reply:
done!